src/components/BreedsList.vue

```diff
@@ -93,30 +86,39 @@ export default {
     },
     getSearchResults(input) {
       this.isLoading = true;
-      setTimeout(() => {
-        if (this.isLoading) {
-          this.isTimeOut = true;
-          this.isLoading = false;
-        }
-      }, 1000);
+      this.message = "Data is loading...";
       this.searchQuery = input;
 
+      if (this.searchQuery.length === 0) {
+        this.isLoading = false;
+      }
+
       searchBreed(this.searchQuery)
         .then((results) => {
-          this.isTimeOut = false;
           this.isLoading = false;
-          this.searchList = results;
           if (results.length > 0) {
+            this.searchList = results.map((breed) => {
+              breed.isActive = false;
+              return breed;
+            });
+
+            this.message = "";
             this.searchList = results;
```

Comment:
tukaj še vedno, še enkrat nastaviš `this.searchList = results;` k tud že 100 vrstici

Reply:
Sem spregledov, popravljeno